### PR TITLE
added: new sharing service: Omnivore

### PR DESCRIPTION
**Pull request by math-GH**
Add a sharing service from FreshRSS to Omnivore

More about Omniore: omnivore.app or https://github.com/omnivore-app/omnivore/

FYI, ping @jacksonh, @sywhb and @satindar

How to test the feature manually:

1. go to config 'Sharing'
2. add Omnivore (create a free account on omnivore.app)
3. share a link from FreshRSS to Omnivore

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/en/gen.php [hunk 1]
```diff
@@ -209,6 +209,7 @@
 		'linkedin' => 'LinkedIn',
 		'mastodon' => 'Mastodon',
 		'movim' => 'Movim',
+		'omnivore' => 'Omnivore',	// TODO
```
Comment (Alkarex) [inline app/i18n/en/gen.php]:
```suggestion
		'omnivore' => 'Omnivore',
```

File: app/i18n/es/gen.php [hunk 2]
```diff
@@ -209,6 +209,7 @@
 		'linkedin' => 'LinkedIn',	// IGNORE
 		'mastodon' => 'Mastodon',	// IGNORE
 		'movim' => 'Movim',	// IGNORE
+		'omnivore' => 'Omnivore',	// TODO
```
Comment (Alkarex) [inline app/i18n/es/gen.php]:
```suggestion
		'omnivore' => 'Omnivore',	// IGNORE
```

And maybe for most languages

File: app/i18n/fr/gen.php [hunk 3]
```diff
@@ -209,6 +209,7 @@
 		'linkedin' => 'LinkedIn',	// IGNORE
 		'mastodon' => 'Mastodon',	// IGNORE
 		'movim' => 'Movim',	// IGNORE
+		'omnivore' => 'Omnivore',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/gen.php]:
```suggestion
		'omnivore' => 'Omnivore',	// IGNORE
```

File: app/i18n/es/gen.php (hunk 2 above)
Comment (math-GH) [reply app/i18n/es/gen.php]: oh yes, you are right. Thanks for pointing to it

Comment (math-GH) [reply app/i18n/es/gen.php]: fixed

File: app/i18n/en/gen.php [hunk 4]
```diff
@@ -209,6 +209,7 @@
 		'linkedin' => 'LinkedIn',
 		'mastodon' => 'Mastodon',
 		'movim' => 'Movim',
+		'omnivore' => 'Omnivore',	// IGNORE
```
Comment (Alkarex) [inline app/i18n/en/gen.php]:
```suggestion
		'omnivore' => 'Omnivore',
```

**Review by ColonelMoutarde — APPROVED** [review]

**Pull request merged**